besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -2041,6 +2041,10 @@ private PrivacyParameters privacyParameters(final KeyValueStorageProvider storag
       if (isPruningEnabled()) {
         throw new ParameterException(commandLine, String.format("%s %s", "Pruning", errorSuffix));
       }
+      if (isGoQuorumCompatibilityMode) {
+        throw new ParameterException(
+            commandLine, String.format("%s %s", "GoQuorum mode", errorSuffix));
+      }
```

Comment:
this works and is the smallest possible change but we could definitely fail earlier